src/Utils/request/uploadFile.ts

```diff
@@ -31,9 +33,15 @@ const uploadFile = async (
         } catch {
           error = xhr.responseText;
         }
-        Notification.BadRequest({ errs: error.errors });
-        reject(new Error("Client error"));
-        reject(new Error("Client error"));
+        const httpError = new HTTPError({
+          message: t("image_upload_validation_error"),
```

Comment:
we can skip this. The http error handler will automatically dispatch toast based on the `cause` object if provided and we are already doing that!

This error message will never be used for display purposes.

See/refer how `HTTPError` is instantiated in other places.
<img width="962" alt="image" src="https://github.com/user-attachments/assets/0a0d39d3-1948-450d-b3c4-413819669c6f" />


```suggestion
          message: "Request failed",
```